test/Microsoft.TestPlatform.PerformanceTests/Microsoft.TestPlatform.PerformanceTests.csproj

```diff
@@ -8,13 +8,12 @@
   <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
   <PropertyGroup>
     <OutputType Condition=" '$(TargetFramework)' == 'net6.0' ">Exe</OutputType>
-    <TargetFrameworks>net6.0;net48</TargetFrameworks>
```

Comment:
Any reason to remove the `net6.0` build?

Reply:
Looking at the tests it seems to be removing the specific conditions, right?

Reply:
Yes, I am not very interested in the test "orchestrator" to run twice. Right now a net48 project is running net451 vstest.console, which runs net6.0 tests. The project did not immediately build when having also net6.0 TFM and I don't want to spend time figuring out how I should do event tracing on net6.0 when I have a working solution on net48.